app/modules/import-export/module.php

```diff
@@ -471,19 +473,19 @@ private function ensure_writing_permissions() {
 
 		// WP Content dir has to be exists and writable.
 		if ( ! $permissions[ Server::KEY_PATH_WP_CONTENT_DIR ]['write'] ) {
-			throw new \Error( self::NO_WRITE_PERMISSIONS_KEY . 'in - ' . Server::KEY_PATH_WP_CONTENT_DIR );
+			throw new \Error( self::NO_WRITE_PERMISSIONS_KEY );
 		}
 
 		// WP Uploads dir has to be exists and writable.
 		if ( ! $permissions[ Server::KEY_PATH_UPLOADS_DIR ]['write'] ) {
-			throw new \Error( self::NO_WRITE_PERMISSIONS_KEY . 'in - ' . Server::KEY_PATH_UPLOADS_DIR );
```

Comment:
Why not show the paths?

Reply:
This will require different error messages in the client. It is possible to see the path by the trace in the system-info/log.